### PR TITLE
readability improvements - examine

### PR DESCRIPTION
**Pull request by erikjanwestendorp**
- [x] rewrite sentence (max 25 words)

### CODE REVIEW

File: 10/umbraco-cms/reference/searching/examine/README.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ _Examine uses Lucene as its search and index engine. Searching using Examine wit
 
 The Examine documentation is found here [https://shazwazza.github.io/Examine/](https://shazwazza.github.io/Examine/) and the source code repository for Examine is here [https://github.com/Shazwazza/Examine](https://github.com/Shazwazza/Examine).
 
-Examine allows you to index and search data quickly. Examine is a library that sits on top of [Lucene.Net](https://lucenenet.apache.org/), a high performance search engine library. Examine provides APIs to make searching and indexing as straight forward as possible. Umbraco provides a further layer on top, UmbracoExamine, that opens up the Umbraco-specific APIs for indexing and searching content and media out of the box.
+Examine allows you to index and search data quickly. Examine is a library that sits on top of [Lucene.Net](https://lucenenet.apache.org/), a high performance search engine library. Examine provides APIs to make searching and indexing as straight forward as possible. UmbracoExamine adds an extra layer to access Umbraco-specific APIs for content and media indexing and searching.
```
Comment (sofietoft) [inline 10/umbraco-cms/reference/searching/examine/README.md:14]: How about we change the order of "indexing and searching" and "content and media"?
Sounds a little more fluid IMO 😄 

Also, Grammarly suggested some changes, that I applied to the sentences before 😅 

```suggestion
Examine allows you to index and search data quickly. Examine is a library that sits on top of [Lucene.Net](https://lucenenet.apache.org/), a high-performance search engine library. Examine provides APIs to make searching and indexing as straightforward as possible. UmbracoExamine adds an extra layer to access Umbraco-specific APIs for indexing and searching content and media.
```

**Review by sofietoft — COMMENTED** [review]
Thanks for the PR @erikjanwestendorp ! 💪 

Just a single suggestion!

**Comment by erikjanwestendorp** [comment]
@sofietoft Thanks for the review, just updated the PR with your suggestion 😄

**Comment by sofietoft** [comment]
Looks great @erikjanwestendorp - Thanks!

I'll merge it right in.

Have a fab' weekend! 🤩

**Pull request merged**